conf/routes

```diff
@@ -1,6 +1,5 @@
 ->            /                                                                                                  health.Routes
 
-GET           /customs-notifications-receiver-stub/admin/metrics                                                 @com.kenshoo.play.metrics.MetricsController.metrics
```

Comment:
Should this be removed?

Reply:
Yes can be removed as bootstrap has been upgraded to > 8.0.0

Reply:
Also this has been mentioned in the play 3 upgrade migration documentation